### PR TITLE
Integration tests - transactions scenarios

### PR DESCRIPTION
**Pull request by miiu96**
Added new integration tests for transactions:
- move balance scenarios
- smart contract deploys and calls
- builtin function calls

### CODE REVIEW

File: integrationTests/multiShard/txScenarios/builtinFunctions_test.go [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+	"testing"
+	"time"
+
+	"github.com/ElrondNetwork/elrond-go/integrationTests"
+	"github.com/ElrondNetwork/elrond-go/process/factory"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestTransaction_TransactionBuiltinFunctionsScenarios(t *testing.T) {
+	t.Parallel()
```
Comment (iulianpascalau) [inline integrationTests/multiShard/txScenarios/builtinFunctions_test.go:17]: I think there is an issue running this in parallel. For integration tests, might remove this.

File: integrationTests/multiShard/txScenarios/moveBalance_test.go [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+	"time"
+
+	"github.com/ElrondNetwork/elrond-go/core/pubkeyConverter"
+	"github.com/ElrondNetwork/elrond-go/integrationTests"
+	"github.com/ElrondNetwork/elrond-go/vm/factory"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestTransaction_TransactionMoveBalanceScenarios(t *testing.T) {
+	t.Parallel()
```
Comment (iulianpascalau) [inline integrationTests/multiShard/txScenarios/moveBalance_test.go:15]: no t.Parallel

File: integrationTests/multiShard/txScenarios/scCalls_test.go [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+	"testing"
+	"time"
+
+	"github.com/ElrondNetwork/elrond-go/integrationTests"
+	"github.com/ElrondNetwork/elrond-go/process/factory"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestTransaction_TransactionSCScenarios(t *testing.T) {
+	t.Parallel()
```
Comment (iulianpascalau) [inline integrationTests/multiShard/txScenarios/scCalls_test.go:18]: t.Parallel

Comment (miiu96) [reply integrationTests/multiShard/txScenarios/scCalls_test.go:18]: done.

File: integrationTests/multiShard/txScenarios/moveBalance_test.go (hunk 2 above)
Comment (miiu96) [reply integrationTests/multiShard/txScenarios/moveBalance_test.go:15]: done.

File: integrationTests/multiShard/txScenarios/builtinFunctions_test.go (hunk 1 above)
Comment (miiu96) [reply integrationTests/multiShard/txScenarios/builtinFunctions_test.go:17]: done.

**Pull request merged**